src/python/pants/build_graph/target.py

```diff
@@ -903,7 +903,7 @@ def create_sources_field(self, sources, sources_rel_path, key_arg=None):
     elif not isinstance(sources, EagerFilesetWithSpec):
       deprecated_conditional(
         lambda: True,
-        '1.11.0.dev0',
+        '1.12.0.dev0',
```

Comment:
Is it possible that deleting these codepaths now would make it easier to make progress on remoting before the next release? Not sure how much dead code is below them.